src/index.ts

```diff
@@ -52,6 +53,7 @@ async function main() {
   await Promise.all(workflow_ids.map(async (workflow_id) => {
     try {
       const { data } = await octokit.actions.listWorkflowRuns({
+        per_page: 100,
```

Comment:
I like the idea of increasing this number. Do you want to submit another PR with just this one change?